constants.go

```diff
@@ -5,6 +5,9 @@ const (
 	ComposerGlobalLayerName   = "composer-global"
 	ComposerPhpIniLayerName   = "composer-php-ini"
 
+	// Autoloader Suffix
+	ComposerAutoloaderSuffix = "PaketoDefaultAutoloaderSuffix"
```

Comment:
Default suffix?

Reply:
?

Reply:
@sophiewigmore Sorry for the confusion. I meant is this a good suffix value?

Reply:
ah. I think this is fine